tasks/localizationtasks.js

```diff
@@ -208,7 +208,18 @@ function dictionaryMapping(file, packageAllKeys = undefined) {
 
 	// Make the new file create on root.
 	// the original directory path was 'vscode-mssql\localization\xliff\<lang>'
-	file.dirname = file.dirname.replace(language, '').replace('localization', '').replace('xliff', '');
+	file.dirname = replaceLastString(file.dirname, `\\${language}`);
```

Comment:
This is not platform-aware, are you sure that the path separators will always be `\`?

Reply:
The separators are unnecessary as I found out